circle.yml

```diff
@@ -22,7 +22,7 @@ dependencies:
 test:
   override:
     - ./gradlew assembleProductionDebug
-    - ./gradlew check
+    - ./gradlew lint
```

Comment:
Can you modify `checkLicense` dependee (https://github.com/DroidKaigi/conference-app-2017/blob/master/app/build.gradle#L302) or run `checkLicense` expressly? 🙇